l10n/sleha/fr-fr/xml/article_installation.xml

```diff
@@ -0,0 +1,748 @@
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Corosync</term>
+    <listitem>
+     <para>Le script configure le système de communication de la grappe.</para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>SBD/surveillance</term>
+    <listitem>
+     <para>Le script vérifie si un système de surveillance existe et vous demande si vous souhaitez configurer un SBD comme mécanisme d&apos;isolation de noeud.</para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Adresse IP virtuelle flottante</term>
+    <listitem>
+     <para>Le script vous demande si vous souhaitez configurer une adresse IP virtuelle pour l&apos;administration de la grappe à l&apos;aide de Hawk2.</para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Pare-feu</term>
+    <listitem>
+     <para>Le script ouvre les ports du pare-feu qui sont nécessaires pour les communications de grappe.</para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Nom de la grappe</term>
+    <listitem>
+     <para>Le script définit un nom pour la grappe. Par défaut, il s&apos;agit de <systemitem>hacluster</systemitem>. Ce nom est facultatif et principalement utile pour les grappes géographiques. En règle générale, le nom de la grappe reflète l&apos;emplacement et facilite la distinction d&apos;un site à l&apos;intérieur d&apos;une grappe géographique.</para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>QDevice/QNetd</term>
+    <listitem>
+     <para>
+      Cette configuration n&apos;est pas traitée ici. Pour utiliser un serveur QNetd, configurez-le avec le script d&apos;amorçage comme décrit dans le document <xref linkend="cha-ha-qdevice"/>.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+ </sect1>
+
+  <sect1 xml:id="sec-ha-inst-quick-installation">
+    <title>Installation de <phrase role="productname"><phrase os="sles">SUSE Linux Enterprise High Availability Extension</phrase></phrase></title>
+    <para>
+      Les paquetages de configuration et de gestion de grappe avec High Availability Extension sont inclus dans le modèle d&apos;installation <literal>haute disponibilité</literal> (appelé <literal>sles_ha</literal> dans la ligne de commande). Ce modèle est disponible uniquement si <phrase role="productname"><phrase os="sles">SUSE Linux Enterprise High Availability Extension</phrase></phrase> a été installé en tant qu&apos;extension de SUSE® Linux Enterprise Server.
+    </para>
+    <para>
+      Pour plus d&apos;informations sur l&apos;installation des extensions, reportez-vous au manuel<link xlink:href="https://documentation.suse.com/sles/html/SLES-all/cha-add-ons.html"><citetitle>Deployment Guide</citetitle> for SUSE Linux Enterprise Server <phrase role="productnumber"><phrase os="sles">15 SP4</phrase></phrase></link> (Guide de déploiement de SUSE Linux Enterprise Server 15 SP4).
```

Comment:
missing space here: 'au manuel<link...'

Reply:
Fix now in XML on my local cloned repo and pushed to 'Vistatec/localisation checkin sleha15sp4' branch in repo.

Reply:
![image](https://user-images.githubusercontent.com/68334239/182836451-37111c84-ba3f-4d99-a0fa-5b2d6e206c5b.png)

Fixed now in the TM

![image](https://user-images.githubusercontent.com/68334239/182836624-35a9ea44-7dcc-43c1-94a3-c35148594d1f.png)